lib/class-wp-block.php

```diff
@@ -169,19 +169,30 @@ public function __get( $name ) {
 	/**
 	 * Generates the render output for the block.
 	 *
+	 * @param array $options         Optional options object.
+	 *        bool  $options.dynamic Optionally set to false to avoid using the block's render_callback.
```

Comment:
I don't believe this is the standard syntax.

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-1-parameters-that-are-arrays

Reply:
Updated.